link/src/main/java/com/stripe/android/link/account/LinkAccountManager.kt

```diff
@@ -217,6 +218,17 @@ internal class LinkAccountManager @Inject constructor(
         }
     }
 
+    /**
+     * Whether the user has logged out from any account.
+     */
+    suspend fun hasUserLoggedOut(email: String?): Boolean {
+        email ?: return true
+        val linkAccount = linkAccount.value ?: return true
+        if (email != linkAccount.email) return true
```

Comment:
Can you improve the readability of this? All of the one-lines + ?: operators are a bit hard to read. I think this could be written as just one conditional statement joined by ors probably?